rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -101,20 +175,519 @@ def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism
         database=database,
         tx_hash=tx_hash,
     )
+    bridge_amount, timestamp = '0.03958480553397407', TimestampMS(1666977475000)
     user_address = optimism_accounts[0]
     expected_events = [
         EvmEvent(
             tx_hash=tx_hash,
             sequence_index=0,
-            timestamp=TimestampMS(1666977475000),
+            timestamp=timestamp,
             location=Location.OPTIMISM,
             event_type=HistoryEventType.WITHDRAWAL,
             event_subtype=HistoryEventSubType.BRIDGE,
             asset=A_ETH,
-            balance=Balance(amount=FVal('0.03958480553397407')),
+            balance=Balance(amount=FVal(bridge_amount)),
             location_label=user_address,
-            notes='Bridge 0.03958480553397407 ETH via hop protocol',
+            notes=f'Bridge {bridge_amount} ETH via Hop protocol',
             counterparty=CPT_HOP,
-            address=string_to_evm_address('0x86cA30bEF97fB651b8d866D45503684b90cb3312'),
-        )]
+            address=ETH_AMM_WRAPPER_OPTIMISM,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(ilter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xc009D690942DbAaC8d5b15B20EFb24fCbFF77Ddd']])
+def test_hop_usdc_bridge(database, ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xdca89892d6a738c2ea278d93a5e1757b2f946f4e24c4aa02d59a0abf6a8e5f4b')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, bridge_amount = TimestampMS(1715038763000), '9006.683634'
+    user_address = ethereum_accounts[0]
```

Comment:
Here and elsewhere please keep these definition in one line as I already asked.

**Fix in the entire file**

```suggestion
    timestamp, bridge_amount, user_address = TimestampMS(1715038763000), '9006.683634', ethereum_accounts[0]
```